### ISSUE TITLE
 ParseDefinitions not supported 'Array' yet.

### ISSUE THREAD

**Issue by 2BAB** [swaggo/swag#299]
**Describe the bug**
I saw there is `{array}` definition in response annotation, but I don't know how to make an array as the parameter. Here is the example:

`// @Param  notification  body  push.ModelList  true  "balabala"`

`type ModelList []Model`

Then I got: `panic: ParseDefinitions not supported 'Array' yet.`

**To Reproduce**
1. Run `swag init`

**Expected behavior**
Support array in parameters annotation.

**Your swag version**
github.com/swaggo/gin-swagger v1.0.0
github.com/swaggo/swag v1.4.0

**Comment by xiaoguiSniper** [comment]
```
type X struct {
  IDs []int64 
}
```
`// @Param Message body api.X true "tips"`
when I use `swag init`,I got this message in api difinition:`Unknown Type:int64`
so,I wonder if any solution for this question? 
or, this is just my fault for `body` type which I want to define an array type.

**Comment by ubogdan** [comment]
Issue #299 supporting array of things is already fixed on #462.
Annotation
`// @Param payload body web.TagList true "description"
`


```
package web

type TagList []Tag

// Tag example
type Tag struct {
	ID   int    `json:"id"`
	Name string `json:"name"`
}
```

Please get latest swagger and open an issue if still not working.
